ibm/service/vpc/data_source_ibm_is_instance_templates.go

```diff
@@ -151,6 +151,7 @@ func DataSourceIBMISInstanceTemplates() *schema.Resource {
 							Computed:    true,
 							Description: "The unique identifier or CRN of the default IAM trusted profile to use for this virtual server instance.",
 						},
+
```

Comment:
Please add the schema for metadata service here

Reply:
Templates is not returning anything please check

 data.ibm_is_instance_templates.instance_templates_data:
data "ibm_is_instance_templates" "instance_templates_data" {
    id = "2023-02-01 12:43:06.68559 +0000 UTC"
}

Reply:
There are multiple places you need to add the logic, ex : line number 770, Please check in all the data sources,
As of now the values are not getting populated in any of the data sources

				if instance.MetadataService != nil {
					d.Set(isInstanceTemplateMetadataServiceEnabled, instance.MetadataService.Enabled)
				}